src/applications/caregivers/utils/constants.js

```diff
@@ -9,7 +9,7 @@ export const ALLOWED_FILE_TYPES = ['pdf', 'jpg', 'jpeg', 'png'];
 
 export const DOWNLOAD_ERRORS_BY_CODE = {
   '5': content['alert-download-message--500'],
-  default: content['alert-download-message--default'],
```

Comment:
`default` is a reserved name, thus we could not de-structure this when utilized in the codebase.